xarray/tests/test_units.py

```diff
@@ -1413,6 +1428,13 @@ def example_1d_objects(self):
         ]:
             yield (self.cls("x", data), data)
 
+    # TODO: remove once pint==0.12 has been released
+    @pytest.mark.xfail(
```

Comment:
Looks like pint 0.12 is out. Shall we remove these xfails now or do you want to do that later?

Reply:
I think something went wrong with the release of `0.12` so I'd like to wait until I got a response from the maintainer

Reply:
:+1: Sounds good. IMO we should merge this.